PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/serviceexplorer/azure/database/actions/SqlServerConnectDataSourceAction.kt

```diff
@@ -27,7 +27,7 @@ import com.microsoft.tooling.msservices.helpers.Name
 import com.microsoft.tooling.msservices.serviceexplorer.azure.database.sqlserver.SqlServerNode
 
 @Name("Connect to server")
-class ConnectServerAction(private val databaseServerNode: SqlServerNode)
+class SqlServerConnectDataSourceAction(private val databaseServerNode: SqlServerNode)
```

Comment:
This rename is not really part of functionality in this PR, but made sense to unify with other action names.